types/es-abstract/es2017.d.ts

```diff
@@ -1,29 +1,118 @@
 import ES2016 = require('./es2016');
 
-interface ES2017 extends Omit<ES2016, 'EnumerableOwnNames' | 'IterableToArrayLike'> {
-    ToIndex(value: unknown): number;
-
-    /** Get own enumerable property keys. */
-    EnumerableOwnProperties(O: object, kind: 'key'): string[];
-
-    /** Get own enumerable property values. */
-    EnumerableOwnProperties<T>(O: { [s: string]: T } | ArrayLike<T>, kind: 'value'): T[];
-
-    /** Get own enumerable property entries. */
-    EnumerableOwnProperties<T>(O: { [s: string]: T } | ArrayLike<T>, kind: 'key+value'): Array<[string, T]>;
-
-    /** Get own enumerable property entries. */
-    EnumerableOwnProperties(O: object, kind: 'key+value'): Array<[string, unknown]>;
-
-    /** Get own enumerable properties. */
-    EnumerableOwnProperties(O: object, kind: 'key' | 'value' | 'key+value'): unknown[];
-
-    /**
-     * @param items The items
-     * @param method The method to use to get the `Iterator`
-     */
-    IterableToList<T>(items: Iterable<T>): T[];
-    IterableToList<O, T>(items: O, method: (this: O) => Iterator<T>): T[];
+interface ES2017 {
+	readonly 'Abstract Equality Comparison': typeof import('./2017/AbstractEqualityComparison');
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/es-abstract/es2017.d.ts;
  Line: 4;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.